### ISSUE TITLE
fail2ban.service file install in rpm packages is broken in 0.10.1

### ISSUE THREAD

**Issue by opoplawski** [fail2ban/fail2ban#1964]
### The issue:

Commit e3b061e94b54067525c5e7f2ac716d1c838c9f20 breaks rpm builds by using the full buildroot path instead of just /usr/ibn

```
+ /usr/bin/python setup.py install -O1 --skip-build --root /export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64
...
Creating build/fail2ban.service (from fail2ban.service.in): @BINDIR@ -> /export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64/usr/bin
...
+ /usr/lib/rpm/check-buildroot
/export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64/usr/lib/systemd/system/fail2ban.service:ExecStart=/export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64/usr/bin/fail2ban-server -xf start
/export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64/usr/lib/systemd/system/fail2ban.service:# ExecStart=/export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64/usr/bin/fail2ban-server -xf --logtarget=stdout start
/export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64/usr/lib/systemd/system/fail2ban.service:ExecStop=/export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64/usr/bin/fail2ban-client stop
/export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64/usr/lib/systemd/system/fail2ban.service:ExecReload=/export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64/usr/bin/fail2ban-client reload
Found '/export/home/orion/redhat/BUILDROOT/fail2ban-0.10.1-1.el7.x86_64' in installed files; aborting
```

**Comment by sebres** [comment]
Should be fixed in eac80966c503b0bc940c119d9a0adafb9ccf50d4.
Do you need a new release of 0.10?

**Comment by sebres** [comment]
Now it was broken without the root-option (unavailable within virtualenv by testing via travis), ultimate fix in 7bf5980defad39c1928209c7636269f712858f44

**Comment by opoplawski** [comment]
Thanks.  I can apply the patch just fine.
